jobs/src/main/scala/dcos/metronome/jobrun/impl/JobRunExecutorActor.scala

```diff
@@ -162,11 +187,11 @@ class JobRunExecutorActor(
   }
 
   def becomeAborting(): Unit = {
-    log.info(s"Execution of JobRun ${jobRun.id} has been aborted")
     // kill all running tasks
     jobRun.tasks.values.filter(t => isActive(t.status)).foreach { t =>
       driverHolder.driver.foreach(_.killTask(t.id.mesosTaskId))
     }
+    log.info(s"Purging $runSpecId")
```

Comment:
s"Purging JobRun $runSpecId"

Reply:
I don't understand why we changed from the above log system...  I don't want to translate the logs (purging means aborting etc.).    And I don't know why we switched from the jobRun.id to the runSpecId.   Can we switch back to the older message please?

Reply:
This was just a debug message, I have removed it... not sure why I have removed the original one :( Thanks for noticing...